react/features/mobile/navigation/components/RootNavigationContainer.js

```diff
@@ -63,15 +68,36 @@ const RootNavigationContainer = ({ dispatch, isWelcomePageAvailable }: Props) =>
                     isWelcomePageAvailable
                         && <>
                             <RootStack.Screen
-                                component = { WelcomePageNavigationContainer }
-                                name = { screen.root }
-                                options = { drawerNavigatorScreenOptions } />
+                                component = { WelcomePage }
+                                name = { screen.welcome.main }
+                                options = { welcomeScreenOptions } />
                             <RootStack.Screen
                                 component = { DialInSummary }
                                 name = { screen.dialInSummary }
                                 options = { dialInSummaryScreenOptions } />
                         </>
                 }
+                <RootStack.Screen
+                    component = { HelpView }
+                    name = { screen.welcome.tabs.settings.help }
```

Comment:
Maybe we should remove "welcome" from the name here, since it no longer applies just to the welcome page.

Reply:
I suggest we leave it as it is because this is part of a different navigator.